[server] `testEmptyProject` of the GitLab context parser fails

The `testEmptyProject` of the GitLab context parser fails:

https://github.com/gitpod-io/gitpod/blob/bc6c9f72c0d0a5a1085e5d37d67c01f1a5af4c31/components/server/src/gitlab/gitlab-context-parser.spec.ts#L531-L548

```
1) TestGitlabContextParser
       testEmptyProject:
     AssertionError: expected { Object (isFile, path, ...) } to have deep property 'ref'
      at TestGitlabContextParser.<anonymous> (src/gitlab/gitlab-context-parser.spec.ts:549:36)
      at Generator.next (<anonymous>)
      at fulfilled (src/gitlab/gitlab-context-parser.spec.ts:19:58)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

The actual result is:
```json
{
  "isFile": false,
  "path": "",
  "title": "gp-test-group/gp-test-empty-project - main",
  "revision": "",
  "repository": {
    "host": "gitlab.com",
    "name": "gp-test-empty-project",
    "owner": "gp-test-group",
    "cloneUrl": "https://gitlab.com/gp-test-group/gp-test-empty-project.git",
    "defaultBranch": "main",
    "private": false
  }
}
```

Opening the empty workspace actually works, though. It's just the test that fails and needs to be fixed.

--- a/components/server/src/gitlab/gitlab-context-parser.spec.ts
+++ b/components/server/src/gitlab/gitlab-context-parser.spec.ts
@@ -531,9 +531,6 @@ class TestGitlabContextParser {
     @test public async testEmptyProject() {
         const result = await this.parser.handle({}, this.user, 'https://gitlab.com/gp-test-group/gp-test-empty-project');
         expect(result).to.deep.include({
-            "ref": undefined,
-            "refType": undefined,
-            "revision": undefined,
             "isFile": false,
             "repository": {
                 "host": "gitlab.com",
@@ -541,9 +538,9 @@ class TestGitlabContextParser {
                 "name": "gp-test-empty-project",
                 "cloneUrl": "https://gitlab.com/gp-test-group/gp-test-empty-project.git",
                 "private": false,
-                "defaultBranch": null
+                "defaultBranch": "main"
             },
-            "title": "gp-test-group/gp-test-empty-project"
+            "title": "gp-test-group/gp-test-empty-project - main"
         })
     }
 

